library/std/convert.qs

```diff
@@ -16,6 +16,34 @@ namespace Microsoft.Quantum.Convert {
         body intrinsic;
     }
 
+    /// # Summary
+    /// Converts a `Result` type to a `Bool` type, where `One` is mapped to
+    /// `true` and `Zero` is mapped to `false`.
+    ///
+    /// # Input
+    /// ## input
+    /// `Result` to be converted.
+    ///
+    /// # Output
+    /// A `Bool` representing the `input`.
+    function ResultAsBool(input : Result) : Bool {
+        return input == One;
+    }
+
+    /// # Summary
+    /// Converts a `Bool` type to a `Result` type, where `true` is mapped to
+    /// `One` and `false` is mapped to `Zero`.
+    ///
+    /// # Input
+    /// ## input
+    /// `Bool` to be converted.
+    ///
+    /// # Output
+    /// A `Result` representing the `input`.
+    function BoolAsResult(input : Bool) : Result {
+        return input ? One | Zero;
```

Comment:
```suggestion
        if input {One} else {Zero}
```
We prefer the code to be expression-based. Therefore, there's no need for 'return' in the last statement. Also, the ternary operator (?|) may be deprecated in future.